core/tests/protractor.conf.js

```diff
@@ -319,9 +323,52 @@ exports.config = {
     // will be available. For example, you can add a Jasmine reporter with:
     //     jasmine.getEnv().addReporter(new jasmine.JUnitXmlReporter(
     //         'outputdir/', true, true));
-    var _ADD_SCREENSHOT_REPORTER = true;
 
-    if (_ADD_SCREENSHOT_REPORTER) {
+    var spw = '';
+    var uniqueString = '';
```

Comment:
Can't we define this inside `specStarted`?

Reply:
moved `uniqueString` inside `specStarted`. Also defined it as a `const`